src/v2/mod.rs

```diff
@@ -612,7 +621,7 @@ fn extract_metadata<T>(response: &tonic::Response<T>) -> endpoints::RPCResult<Bl
 ///
 /// The main reason for needing this is that in proto3 all fields are optional,
 /// so it is up to the application to validate inputs if they are required.
-trait Require<E> {
+pub trait Require<E> {
```

Comment:
So should this.

Reply:
pub(crate) I assume would be ok? It's so I don't have to create a new module for the new types.

Reply:
Moved to v2 mod.